worlds/shivers/__init__.py

```diff
@@ -0,0 +1,184 @@
+from .Items import item_table, ShiversItem
+from .Rules import set_rules
+from BaseClasses import Item, Tutorial, Region, Location
+from Fill import fill_restrictive
+from worlds.AutoWorld import WebWorld, World
+from . import Constants, Rules
+from .Options import ShiversOptions
+
+
+class ShiversWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Shivers Setup Guide",
+        "A guide to setting up Shivers for Multiworld.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["GodlFire, Mathx2"]
+    )]
+
+class ShiversWorld(World):
+    """ 
+     Shivers is a horror themed point and click adventure. Explore the mysteries of Windlenot's Museum of the Strange and Unusual.
```

Comment:
minor nitpick, extra space.

```suggestion
    Shivers is a horror themed point and click adventure. Explore the mysteries of Windlenot's Museum of the Strange and Unusual.
```

Reply:
Extra space removed